index.html

```diff
@@ -885,8 +885,8 @@
 * their phone number,
 * their location data,
 * an online identifier such as email or IP addresses, or
-* factors specific to their physical, physiological, genetic, mental, economic, cultural, or social
-[=identity=].
+* factors specific to their physical, physiological, genetic, mental, economic,
+cultural, social, behavioral, or browser [=identity=].
```

Comment:
```suggestion
cultural, social or behavioral [=identity=].
```

Reply:
@kdenhartog by "browser" did you mean cookie-based things or more like fingerprinting? If cookie-based, it feels like it's already captured by the identifier bullet but we could also add a bullet for fingerprinting data (while also mentioning unsanctioned tracking).

Reply:
Yeah I was meaning more like the browsers fingerprint. So more in line with if someone were to run some subset of the https://www.amiunique.org scan and assign it an identifier is the main style of tracking I wanted to call out further.

I called this the browser's identity because it's characteristics about the browser itself, not about me, but I think "browser's fingerprint identity" may be a better term to use here to further clarify.